src/main/java/me/lokka30/phantomworlds/commands/phantomworlds/subcommands/TeleportSubcommand.java

```diff
@@ -38,68 +45,47 @@
      */
     @Override
     public void parseCommand(@NotNull PhantomWorlds main, CommandSender sender, Command cmd, String label, String[] args) {
-        if (!sender.hasPermission("phantomworlds.command.phantomworlds.teleport")) {
+    	String subCommand = args[0].toLowerCase(Locale.ROOT);
```

Comment:
Note to self: rename this variable to `subCommandLabel` - probably better than `subCommand` since that variable could falsely hint it is of the type `SubCommand`. Totally not necessary ... just nitpicking .😛